shell/src/main/java/alluxio/cli/fsadmin/doctor/ConfigurationCommand.java

```diff
@@ -0,0 +1,90 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.cli.fsadmin.doctor;
+
+import alluxio.client.MetaMasterClient;
+import alluxio.wire.ConfigCheckReport;
+import alluxio.wire.ConfigCheckReport.ConfigStatus;
+import alluxio.wire.InconsistentProperty;
+import alluxio.wire.Scope;
+
+import java.io.IOException;
+import java.io.PrintStream;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Prints runtime configuration information.
+ */
+public class ConfigurationCommand {
+  private MetaMasterClient mMetaMasterClient;
+  private PrintStream mPrintStream;
+
+  /**
+   * Creates a new instance of {@link ConfigurationCommand}.
+   *
+   * @param metaMasterClient client to get server-side configuration report information
+   * @param printStream stream to print configuration errors/warnings to
+   */
+  public ConfigurationCommand(MetaMasterClient metaMasterClient, PrintStream printStream) {
+    mMetaMasterClient = metaMasterClient;
+    mPrintStream = printStream;
+  }
+
+  /**
+   * Runs doctor configuration command.
+   *
+   * @return 0 on success, 1 otherwise
+   */
+  public int run() throws IOException {
+    ConfigCheckReport report = mMetaMasterClient.getConfigReport();
+    ConfigStatus configStatus = report.getConfigStatus();
+    if (configStatus.equals(ConfigStatus.PASSED)) {
```

Comment:
safer to use
`configStatus == ConfigStatus.PASSED` in case `configStatus` is null

Reply:
Changed, thanks.